buddyboss/buddyboss-platform
# 1004 - Fixed the issue of send empty messages and extra spaces

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #1004 

https://github.com/buddyboss/buddyboss-theme/pull/860

### How to test the changes in this Pull Request:

1. Go to messages and try to sent a message to some user
2. Press a space button (one or several times) and send the message
3. Try to send a " test test " message (with using a lot of spaces, GitHub don't allow to do it here)
Also, Fix another issue as below
Steps to reproduce the behaviour:
1. Go to messages and try to sent a message to some user
2. Try to send some link ("https://google.com " for example) with some spaces in the end
3. You will see extra &nbsp; symbol at the end of the link

### Proof Screenshots or Video

https://www.loom.com/share/d57c98517c5c449c901e7d3a90228a69

### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
